src/main/java/org/isf/video/gui/PatientPhotoPanel.java

```diff
@@ -70,6 +70,8 @@ public class PatientPhotoPanel extends JPanel {
 
 	private final PhotoboothPanelPresentationModel photoboothPanelPresentationModel;
 
+	private static final int MAXPROFPICFILESIZEBYTES = 4194304;
```

Comment:
how this limit is determinate?  it is a DB restriction/limitation? could it be defined as an external property ?

Reply:
at the moment it is just to simplify the use of the system, without increasing the number of parameters users have to dial with, but it will documented (see. https://github.com/informatici/openhospital-doc/pull/155)

Reply:
fair enough